HousingSearchListener/V1/Interfaces/ElasticSearchUpdater.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Amazon.Lambda.SNSEvents;
+using Amazon.Lambda.SQSEvents;
+using HousingSearchListener.Gateways;
+using HousingSearchListener.Infrastructure;
+using HousingSearchListener.V1.Domain;
+using Microsoft.AspNetCore.WebUtilities;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace HousingSearchListener.V1.Interfaces
+{
+    public class ElasticSearchUpdater : BaseFunction, IElasticSearchUpdater
+    {
+        private IESPersonFactory _esPersonFactory;
+        private IElasticSearchHelper _esHelper;
+        private IHttpHandler _httpHandler;
+        private IPersonMessageFactory _personMessageFactory;
+
+        public ElasticSearchUpdater(IServiceCollection services) : base(services)
```

Comment:
You don't need to pass in the `IServiceCollection`, the `BaseFunction` can just create one itself.

Reply:
It's for integration test purposes.

Reply:
Oh my bad - you are right. Correcting!